pkg/resources/function_acceptance_test.go

```diff
@@ -31,39 +31,39 @@ func TestAcc_Function(t *testing.T) {
 			{
 				Config: functionConfig(functName, acc.TestDatabaseName, acc.TestSchemaName),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "name", functName),
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "comment", "Terraform acceptance test"),
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "statement", expBody2),
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "arguments.#", "1"),
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "arguments.0.name", "ARG1"),
-					resource.TestCheckResourceAttr("snowflake_function.test_funct", "arguments.0.type", "VARCHAR"),
+					resource.TestCheckResourceAttr("snowflake_function.test_function", "name", functName),
```

Comment:
We discussed that with SDK migration we are adding the missing tests. Can you add the one for scala? Can you add update/import steps?

Reply:
done